.travis.yml

```diff
@@ -0,0 +1,30 @@
+sudo: false
+language: go
+go_import_path: go.uber.org/goleak
+
+go:
+  # TODO(prashant): Enable once we have fixed Linux issues with old versions.
+  #- 1.6
+  #- 1.7
+  #- 1.8
+  - 1.9
+
+matrix:
+  include:
+  - go: 1.9
+    env: NO_TEST=yes COVERAGE=yes LINT=yes
```

Comment:
no testing on 1.9?

Reply:
I thought `include` would add a new entry, but looks like it updates it. Cleaned up the env vars, and now have everything enabled for 1.9.